packages/config-plugins/src/ios/Updates.ts

```diff
@@ -48,20 +59,27 @@ export const withUpdates: ConfigPlugin<{ expoUsername: string | null }> = (
   { expoUsername }
 ) => {
   return withExpoPlist(config, config => {
-    config.modResults = setUpdatesConfig(config, config.modResults, expoUsername);
+    const expoUpdatesPackageVersion = getExpoUpdatesPackageVersion(config.modRequest.projectRoot);
```

Comment:
Since we are getting the `projectRoot` from an attribute of `config`, can we move `getExpoUpdatesPackageVersion` inside of `getUpdatesCheckOnLaunch`?

Reply:
Having it as a separate parameter makes the unit tests a lot nicer since we can easily check the different cases without having to mock the filesystem.